lib/enumerated_constants.rb

```diff
@@ -15,12 +17,12 @@ def all
   # @param name [Symbol,String] The name of the constant you don't want
   # @return [Array] @all except the value of that constant
   def except(*names)
-    const_names =
     names = names.map(&:upcase).map(&:to_sym)
     values = names.map do |name|
       begin
         const_get(name)
       rescue NameError
+        raise ArgumentError, "Constant #{name} does not exist"
```

Comment:
Sounds reasonable. It's a relatively breaking change that warrants a version bump.